src/SFA.DAS.EAS.Api/Orchestrators/StatisticsOrchestrator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using MediatR;
+using SFA.DAS.EAS.Account.Api.Types;
+using SFA.DAS.EAS.Application.Queries.GetTheRdsRequiredStatistics;
+using SFA.DAS.NLog.Logger;
+
+namespace SFA.DAS.EAS.Account.Api.Orchestrators
+{
+    public interface IStatisticsOrchestrator
+    {
+        Task<OrchestratorResponse<RdsRequiredStatisticsViewModel>> GetTheRdsRequiredStatistics();
+    }
+
+    public class StatisticsOrchestrator : IStatisticsOrchestrator
```

Comment:
We're moving away from using orchestrators so feel free to not use them if you prefer.